### PR TITLE
[lock/unlock] _receiveLiquidity starting point, arb adapter fixes

### PR DESCRIPTION
**Pull request by RensR**
## Motivation

The Rebalancer contract needs to be able to inject liquidity into token pools in addition to sending liquidity from them.

## Solution

This PR implements a starting point for _receiveLiquidity which will be filled in later after some details are figured out. In addition, it fixes the bridge adapter interface's naming and the Arbitrum L1 Bridge Adapter in particular. See https://github.com/smartcontractkit/ccip/pull/456 and https://github.com/smartcontractkit/ccip/pull/451 for more details on the bridge adapter changes.

### CODE REVIEW

**Review by makramkd — APPROVED** [review]

File: contracts/src/v0.8/rebalancer/bridge-adapters/ArbitrumL1BridgeAdapter.sol [hunk 1]
```diff
@@ -107,19 +125,20 @@ contract ArbitrumL1BridgeAdapter is IL1BridgeAdapter {
     bytes data;
   }
 
-  /// @param l2Sender sender if original message (i.e., caller of ArbSys.sendTxToL1)
-  /// @param l1Receiver destination address for L1 contract call
-  function finalizeWithdrawERC20FromL2(
-    address l2Sender,
-    address l1Receiver,
+  /// @notice Finalize an L2 -> L1 transfer.
+  /// @param remoteSender sender if original message (i.e., caller of ArbSys.sendTxToL1)
```
Comment (matYang) [inline contracts/src/v0.8/rebalancer/bridge-adapters/ArbitrumL1BridgeAdapter.sol:129]: nit: seems like a typo: `sender of`

File: contracts/src/v0.8/rebalancer/bridge-adapters/ArbitrumL1BridgeAdapter.sol [hunk 2]
```diff
@@ -54,36 +54,54 @@ contract ArbitrumL1BridgeAdapter is IL1BridgeAdapter {
...
-    i_l1GatewayRouter.outboundTransferCustomRefund{value: msg.value}(
-      l1Token,
-      recipient,
-      recipient,
-      amount,
-      MAX_GAS,
-      GAS_PRICE_BID,
-      abi.encode(MAX_SUBMISSION_COST, bytes(""))
-    );
+    // TODO: return data bombs?
```
Comment (matYang) [inline contracts/src/v0.8/rebalancer/bridge-adapters/ArbitrumL1BridgeAdapter.sol:92]: Aren't the routers natively managed by Arb? Curious why ret data bomb would be a concern

**Review by matYang — APPROVED** [review]

File: contracts/src/v0.8/rebalancer/bridge-adapters/ArbitrumL1BridgeAdapter.sol (hunk 2 above)
Comment (makramkd) [reply contracts/src/v0.8/rebalancer/bridge-adapters/ArbitrumL1BridgeAdapter.sol:92]: That's a good question, I'm not 100% sure. What I've seen generally is one should be cautious calling out to external contracts.

Comment (RensR) [reply contracts/src/v0.8/rebalancer/bridge-adapters/ArbitrumL1BridgeAdapter.sol:92]: Yeah I think it's safe but worth explicitly checking if there is any risk

**Pull request merged**